### ISSUE TITLE
Question about GET request to API (did I do it wrong?)

### ISSUE THREAD

**Issue by DragonOsman** [boostorg/beast#1038]
As the title says, I need to ask if I'm doing something wrong with the way I'm sending the request to the currency API I want to get rates from.  

The request is sent since `ec`'s message string is reporting success.  But now I'm getting a "Bad request" error from the API itself.  I've checked the URL and it seems to check out.  So I need to ask: What could I be doing wrong with my request?

I made another Gist.  Link: https://gist.github.com/DragonOsman/7bbdb049ea2a77173c5dafbecd5f1be2

Thanks in advance for any help.

**Comment by DragonOsman** [comment]
Hey; sorry for the double post, but I need to give an update here.

I tried to send the request manually to the same URL and port, and it worked that way.  I did it like this: `apilayer.net:80/` (and then the whole target).  

So why won't it work when I do it in the app?  Could it be that I shouldn't use port 80 for a client request while also running the server on port 8080 (or is that not a problem?)?
